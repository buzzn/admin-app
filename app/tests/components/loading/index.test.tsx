import React from 'react';
import renderer from 'react-test-renderer';
import Loading from 'components/loading';

describe('loading tests', () => {
  test('default unit (rem)', () => {
    const tree = renderer
      .create(<Loading {...{ minHeight: 20 }} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  test('px', () => {
    const tree = renderer
      .create(<Loading {...{ minHeight: 20, unit: 'px' }} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
