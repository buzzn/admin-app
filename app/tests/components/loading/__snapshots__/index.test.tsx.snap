// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loading tests default unit (rem) 1`] = `
<div
  style={
    Object {
      "display": "flex",
      "flexDirection": "column",
      "justifyContent": "center",
      "minHeight": "20rem",
      "width": "100%",
    }
  }
>
  <div
    className="la-ball-atom la-dark"
    style={
      Object {
        "margin": "0 auto",
      }
    }
  >
    <div />
    <div />
    <div />
    <div />
  </div>
</div>
`;

exports[`loading tests px 1`] = `
<div
  style={
    Object {
      "display": "flex",
      "flexDirection": "column",
      "justifyContent": "center",
      "minHeight": "20px",
      "width": "100%",
    }
  }
>
  <div
    className="la-ball-atom la-dark"
    style={
      Object {
        "margin": "0 auto",
      }
    }
  >
    <div />
    <div />
    <div />
    <div />
  </div>
</div>
`;
